### ISSUE TITLE
Networking for VMs with static IPs.

### ISSUE THREAD

**Issue by taboche** [vexxhost/migratekit#34]
I got the migration working with Ubuntu 22.04. However, our VMware VMs are configured with static IPs, and we would like to use neutron DHCP IP assignment in OpenStack. The migrated VMs are not getting IPs because of the static networking config. Is it possible to run virt-customize type commands to manipulate the network settings?

**Comment by mnaser** [comment]
@taboche if you put the same exact IP configuration as the static configured one, wouldn't the VM just boot fine with no problems?

**Comment by taboche** [comment]
@mnaser That would have been great. In my case, I cannot use the same subnet on OpenStack side and also OpenStack IP range are in different subdomain so I needed to make that change.

**Comment by mnaser** [comment]
I don't think we have an easy way to implement this, if there is something in the virt-v2v toolkit that can do this, perhaps we can tap into it, but we lean heavily on virt-v2v for playing around the actual state of the VM.

**Comment by taboche** [comment]
@mnaser I had some success with virt-v2v firstboot script feature. This script has to be smart enough to take care of all OS distro though.
